lib/litle.rb

```diff
@@ -27,6 +27,14 @@
 module ActiveMerchant #:nodoc:
   module Billing #:nodoc:
     class LitleGateway < Gateway
+      def initialize(options={})
+        if options.has_key?(:test_url)
```

Comment:
> If I remove the defaults from `litle.yml` and make this change, then I get the same error as Travis. The reason is that the `commit` method in `litle.rb` accesses the `test_url` and passes it to `ssl_post`, but the url will be nil

I don't understand -- shouldn't it be set by [ActiveMerchant](https://github.com/activemerchant/active_merchant/blob/master/lib/active_merchant/billing/gateways/litle.rb#L8)?

How did it work before?